features/utils/apiUtils.js

```diff
@@ -1,14 +1,69 @@
-const axios = require('axios');
```

Comment:
Better to create a separate file for the reqres client. e.g. `reqres_client.js`

Reply:
The suggestion to create a separate file for the reqres_client.js is from the best practices in software development:

- Modularity: Keeping the code modular by separating concerns into different files makes it easier to maintain and understand. Each file can focus on a specific aspect of the application, reducing complexity.

- Reusability: By isolating the reqres client in its own file, you can reuse the same client across different parts of your application without duplicating code. This promotes DRY (Don't Repeat Yourself) principles.

- Testing: It's easier to test individual components when they are separated into different files. A dedicated file for the reqres client means you can write and run tests specifically for your API interactions without interference from other parts of your application.

- Collaboration: When working in a team, having separate files can reduce merge conflicts and make it clearer who is working on what. It's easier to manage version control when changes are isolated to specific areas.

By separating the reqres_client.js into its own file, you’re structuring your codebase in a way that is scalable, maintainable, and easier to navigate. This approach can lead to better software architecture and easier management of the project as it grows.